vtexdocs/devportal
# Feature/Table of Contents

#### What is the purpose of this pull request?

To add a Table of Contents to the API Guide documentation pages.

Documented [here](https://www.notion.so/vtexhandbook/Adicionar-Table-of-Contents-nas-p-ginas-de-API-Guides-5937b40c11d44c2096e3a7276ca5cdd0)

#### What problem is this solving?

The lack of a component for rendering a Table of Contents. The Table of Contents helps the reader keep track of where they are in the article and navigate to specific sections faster.

#### How should this be manually tested?

Go to this page and try scrolling up and down. The Table of Contents should update to show in which part of the article you're at. Try clicking in the links inside the Table of Contents, you should be taken to the corresponding section of the documentation and the Table of Contents should update accordingly (do keep in mind that clicking the link to a heading that is too close to another one may have the effect of making the other heading show as active in the Table of Contents).

#### Screenshots or example usage

#### Types of changes
